src/training/claim_detection.py

```diff
@@ -8,6 +10,8 @@
 from tqdm import tqdm
 from transformers import AutoTokenizer
 
+grandparent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))  #get the "src" directory
```

Comment:
why did you need this? Maybe you were running this file as

python path/to/file.py?

Instead of this you can run

python -m path.to.file

Then you don't need it. All the import paths in the repo refer to the base directory of the repo, and from that directory you can run everything in the repo, too. I don't have a strong opinion on this but I think this is the preferred way.

https://stackoverflow.com/questions/22241420/execution-of-python-code-with-m-option-or-not

Reply:
OK, so we can ignore these two lines